### PR TITLE
Enable expeditor and pipeline

### PR DESCRIPTION
**Pull request by scotthain**
Signed-off-by: Scott Hain <shain@chef.io>

This is the first step in upgrading the chef-web-docs internal deploy process!

### CODE REVIEW

File: .expeditor/verify.pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- label: "[chef/chef-web-docs] build"
```
Comment (tduffield) [inline .expeditor/verify.pipeline.yml]: the `[]` bit is unnecessary

File: .expeditor/config.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+slack:
+  notify_channel: releng-notify
+
```
Comment (schisamo) [inline .expeditor/config.yml:3]: You'll want to add:

```yaml
github:
  delete_branch_on_merge: true
```

**Review by schisamo — CHANGES_REQUESTED** [review]

File: .expeditor/verify.pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    command:
```
Comment (tduffield) [inline .expeditor/verify.pipeline.yml]: this should all be at the same indent level as label

**Review by tduffield — CHANGES_REQUESTED** [review]

File: .expeditor/config.yml (hunk 2 above)
Comment (scotthain) [reply .expeditor/config.yml:3]: Done

**Review by schisamo — APPROVED** [review]

**Review by tduffield — APPROVED** [review]

**Pull request merged**